app/src/main/java/com/infomaniak/drive/data/cache/DriveInfosController.kt

```diff
@@ -141,16 +141,20 @@ object DriveInfosController {
 
             drive?.categories?.let {
                 val categories = it.where()
-                    .`in`(Category::id.name, ids)
```

Comment:
val categories = it.where()
                    .apply { fileCategoriesIds?.let { `in`(Category::id.name, fileCategoriesIds) } }
                    .findAll()